### ISSUE TITLE
Update statement for renaming label needs to be qualified with creation time

### ISSUE THREAD

**Issue by luuse** [hi3g/freki#48]
In `CassandraStore#renameLabel` we don't qualify the update statement with the creation time of the ID and since `creationTime` is part of the primary key the query fails.

Running this query in `cqlsh` shows it clearly:

```
UPDATE id_to_name SET name='newname' WHERE label_id=7480080820347656939 AND type='tagk';
```

**Comment by luuse** [comment]
#50 adds tests that are currently ignored that shows this.

**Comment by luuse** [comment]
Fixed in #87
